tests/subsys/bluetooth/mesh/scheduler_model/common/sched_test.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <scheduler_internal.h>
+#include <time_util.h>
+
+#define DELTA_TIME   1
+#define SUBSEC_STEPS 256
+
+/* item of struct tm */
+#define ISTM(year, month, day, hour, minute, second) \
```

Comment:
I think we can afford another 3 characters here and call this "TM_INIT" or "TM_ITEM".